src/Options/Generic.hs

```diff
@@ -826,18 +859,68 @@ getRecordPure
     -- ^ Command-line arguments
     -> Maybe a
 getRecordPure args = do
-    let prefs = Options.ParserPrefs
-            { prefMultiSuffix     = "..."
-            , prefDisambiguate    = False
-            , prefShowHelpOnError = False
-            , prefBacktrack       = True
-            , prefColumns         = 80
-#if MIN_VERSION_optparse_applicative(0,13,0)
-            , prefShowHelpOnEmpty = False
-#else
-#endif
-            }
     let header = Options.header ""
     let info   = Options.info parseRecord header
     let args'  = map Data.Text.unpack args
-    Options.getParseResult (Options.execParserPure prefs info args')
+    Options.getParseResult (Options.execParserPure ogPrefs info args')
+
+-- | @optparse-generic@'s flavor of options
+ogPrefs :: Options.ParserPrefs
```

Comment:
Perhaps rename this to `defaultParserPrefs`.  There is no need for the `og` prefix since the user can qualify the module import if they need to disambiguate the name